source/install/install-rhel-66-mysql.rst

```diff
@@ -0,0 +1,66 @@
+..  _install-rhel-66-mysql:
+
+Installing MySQL Database Server
+================================
+
+Install and set up the database for use by the Mattermost server. You can install either MySQL or PostgreSQL. To install PostgreSQL, see :ref:`install-rhel-66-postgresql`.
```

Comment:
I'm not sure an "Installing Database Server" section would help. In the monolith style, you'd still have installing MySQL followed by installing PostgreSQL, and it's already noted in several places that you need install only one. Plus, I think when someone sees both databases in the ToC or anywhere else,  they don't immediately think that both databases need to be installed; it just doesn't make sense that two databases are required.

I think an extra section adds complexity to the docs without having a significant impact on clarity and understandability.